connaisseur/alert.py

```diff
@@ -0,0 +1,177 @@
+import json
+import logging
+import os
+from datetime import datetime
+
+import requests
+from jinja2 import Template, StrictUndefined
+
+from connaisseur.exceptions import AlertSendingError, ConfigurationError
+from connaisseur.image import Image
+from connaisseur.mutate import get_container_specs
+
+
+class Alert:
+    """
+    Class to store image information about an alert as attributes and a sending functionality as method.
+    Alert Sending can, depending on the configuration, throw an AlertSendingError causing Connaisseur
+    responding with status code 500 to the request that was sent for admission control,
+    causing a Kubernetes Error event.
+    """
+
+    connaisseur_pod_id: str
+    cluster: str
+    timestamp: int
+
+    request_id: str
+    images: list
+
+    template: str
+    receiver_url: str
+    payload: dict
+    headers: dict
+
+    alert_message: str
+    priority: int
```

Comment:
only the variables `receiver_url`, `template`, `payload` and `header` are actually being used. also `context`, `admission_request` and `throw_if_alert_sending_fails` are missing. the rest should be deleted.